lms/djangoapps/program_enrollments/models.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+Django model specifications for the Program Enrollments API
+"""
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import ugettext as _
+from model_utils.models import TimeStampedModel
+from simple_history.models import HistoricalRecords
+
+
+class ProgramEnrollments(TimeStampedModel):  # pylint: disable=model-missing-unicode
+    """
+    This is a model for Program Enrollments from the registrar service
+
+    .. pii:
+    """
+    STATUSES = (
+        ('enrolled', _(u'enrolled')),
+        ('pending', _(u'pending')),
+        ('suspended', _(u'suspended')),
+        ('withdrawn', _(u'withdrawn')),
+    )
+
+    class Meta(object):
+        app_label = "program_enrollments"
+
+    user = models.ForeignKey(
+        User,
+        null=True,
+        blank=True
+    )
+    email = models.EmailField()
```

Comment:
This field also needs to be `null=True` and `blank=True` because some school may not be able to send us this information